main-story.html

```diff
@@ -4,3 +4,6 @@
 A romantic biologist takes a stroll to the outskirts of the city. Not necessarily for pleasure, but for study. The biologist finds a good assortment of flowers and plants of all kinds, and becomes infatuated. But all of a sudden, they begin to mutate. The biologist, shocked with terror, knows that there was only one appropriate thing to do here: take three photos. And because she valued her life, she also ran away afterwards. But the mutated plants chase her, using their leaves or vines as legs, and almost catch her.
 
 Out of the blue, she sees the town’s sorcerer floating in on a platform of yellow haze. The sky darkens and the clouds grow thick and dense. The biologist doesn’t look back as she weaves through the tall grass. Missing her footing as she hops over a fallen branch, she trips and lies victim to the mutant plants. At the perfect time, the sorcerer casts a shield over her and calls down thunder upon the mutant plants. The mutants lay motionless and the romantic biologist is saved. 	
+
+
+The biologist stands up and thanks the sorcerer for his bravery. As the biologist is talking the sorcerer notices a cute but mischievous programmer scanning the dead mutants.  The sorcerer, intrigued by the programmer approaches him and asks, “what are you doing mate?”  the programmer answers in an arrogant way, “I am having a milkshake “mate.”” The biologist peaks into the reading the programmer was getting and says, “wow the PH levels are off the charts.” The sorcerer uses a scan spell on the biologist, programmer, and plants. The sorcerer askes the group “do you guys know what is going on guys?” This is not from my realm of expertise. Why are you scanning all of the plants?” he asked the software developer admiring his wits and scanning around for more mutant plants.
```

Comment:
Sorcerers smell something wrong about the programmer since his behavior is different to sorcerer and biologist. The conversation between sorcerer and programmer had arguments also the way programmer had arrogantly replied to sorcerer. Sorcerer  gets jealous because the biologist gives a compliment to the programmer silently without others knowing. Sorcerer worries that the biologist is blind because he trusts the programmer without knowing the intention of the programmer’s idea. But the biologist complimented him for the fake action did before. Sorcerers watch from hidden places to see what biologists compliment programmers.  Sorcerer gets mad at one point of time because he feels that it is not safe until knowing the complete intention of the programmer. Sorcerers  throw fireballs at the biologist to break the conversation between them. Once the programmer left, the sorcerer revealed the truth and suspected a biologist about the programmer. They decided to observe the programmer's behavior.